### PR TITLE
Modify artik053/iotivity defconfig

### PR DESCRIPTION
**Pull request by HONGCHAEHEE**
This patch is to update the iotivity config as TizenRT wireless configuration changed.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [e333e9a3329e390ad94f8ef84956e9c20df9a5b0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [e333e9a3329e390ad94f8ef84956e9c20df9a5b0] -

**Comment by seinfra** [comment]
Target : [39b2d0df7331d91602cefac9f30f1b063bd43ddf] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [39b2d0df7331d91602cefac9f30f1b063bd43ddf] - Code Rule Check OK.

File: build/configs/artik053/iotivity/defconfig [hunk 1]
```diff
@@ -568,20 +559,6 @@ CONFIG_UART4_2STOP=0
 #
 CONFIG_DRIVERS_WIRELESS=y
 CONFIG_SCSC_WLAN=y
-# CONFIG_SLSI_RX_PERFORMANCE_TEST is not set
```
Comment (sunghan-chang) [inline build/configs/artik053/iotivity/defconfig]: Maybe this is wrong. You might keep these configs.

**Comment by seinfra** [comment]
Target : [b303a4cb89703c5c780569c50598f1ee2ac351a3] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b303a4cb89703c5c780569c50598f1ee2ac351a3] - Code Rule Check OK.

**Pull request merged**